cert/tls.go

```diff
@@ -26,20 +26,26 @@ var (
 // LoadCert loads a certificate and its corresponding private key from the PEM
 // files indicated and returns the certificate in the two formats it is most
 // commonly used.
-func LoadCert(tlsCertPath, tlsKeyPath string) (tls.Certificate,
-	*x509.Certificate, error) {
+func LoadCert(certPath, keyPath string) (tls.Certificate, *x509.Certificate,
+	error) {
 
-	certData, err := tls.LoadX509KeyPair(tlsCertPath, tlsKeyPath)
+	// The certData returned here is just a wrapper around the PEM blocks
+	// loaded from the file. The PEM is not yet fully parsed but a basic
+	// check is performed that the certificate and private key actually
+	// belong together.
```

Comment:
Why not add comments in commit where this code is introduced? (assuming it is indeed new)